### DISCUSSION TITLE
Community cards should add the icons of their required expansions

### DISCUSSION THREAD

**Discussion by kevinb9n** [bafolts/terraforming-mars#2131] [Ideas]
... for the same reasons.  Not a big deal, just seems like a good practice.

**Comment by kevinb9n** [comment]
I may be confused, perhaps this was only on the /cards-list that I saw the "this requires turmoil" / "this requires venus" icons.  Either way though, good to have imho.

**Comment by kberg** [comment]
This seems done.

**Comment by kberg** [comment]
Now the icons are on the cards themselves.

![image](https://github.com/user-attachments/assets/ff48adc2-3890-464f-9c5e-1e5b7a6d9356)
